### ISSUE TITLE
Get camera position when it changes

### ISSUE THREAD

**Issue by nyoung697** [Kitware/react-vtk-js#72]
Is there a way to pass in a callback function that receives the camera position after the user is finished panning or rotating?

I have a `<View />`, and I see there are props to set the initial cameraViewUp, and cameraPosition. I would like to save the state of the view, so when the page is refreshed, I can restore it to how the user had it.

**Comment by floryst** [comment]
You should be able to do the following:

```
const viewRef = React.createRef();
...
return <View ref={viewRef} ... />
```

**Comment by nyoung697** [comment]
Oh nice! That works!  I thought I had tried that! haha... Thank you!

**Comment by nyoung697** [comment]
This seems to work on version `1.11.2` but not on `1.12.6`

**Comment by floryst** [comment]
Ah, it might be `viewRef.current.viewRef.current`. I had made changes to support multi-renderer configurations, which resulted in a change of API. `ViewContainer` now wraps `View`, resulting in an additional layer of indirection through `viewContiainer.viewRef.current`.

**Comment by nyoung697** [comment]
I get an error in the console when trying to pass a ref into the View component using version `1.12.6`.

<img width="708" alt="Screen Shot 2022-07-09 at 8 11 53 AM" src="https://user-images.githubusercontent.com/12564184/178109490-2e920e24-323e-4cc2-ab3f-045fd174edc2.png">

**Comment by floryst** [comment]
This should be fixed in #73. We were not forwarding refs properly. Additionally, I made it such that refs forward through the ViewContainer to the View, so `viewRef.current` should point directly to the inner View.

**Comment by floryst** [comment]
I merged #73, so you can now access the inner View component properly using a ref.

**Comment by nyoung697** [comment]
Thank you!
